integration/fuse/bin/alluxio-fuse

```diff
@@ -122,6 +128,14 @@ case $1 in
       mount_fuse $2 $3
       exit $?
     fi
+    if [ $# -eq 4 ] && [ $3 == "-o" ]; then
```

Comment:
This means `-o` can only come at the end of the command? Is that how we handle options for other shell commands?

Reply:
The other shells use `getopts` to parse input arguments. If we use `getopts`, it will be long but also only support `./alluxio-fuse mount -o mount_option local_folder alluxio_folder`.
`mount_option` must follow `mount` to be supported by `getopts`.
```
case $1 in
  mount)
    shift
    while getopts "o:" option; do
      case "${option}" in
        o)
          mount_option=${OPTARG}
          echo "get the mount option"
          ;;
        *)
          echo -e "${USAGE}" >&2
          exit 1
          ;;
      esac
    done
    shift $((${OPTIND} - 1))
    if [[ $# -eq 1 ]]; then
      mount_fuse $1 /
      exit $?
    fi
    if [[ $# -eq 2 ]]; then
      mount_fuse $1 $2
      exit $?
    fi
    echo -e "Usage\n\t$0 mount mount_point [alluxio_path]\n\t alluxio_path is default to root" >&2
    exit 1
    ;;
```